RadarSDK/RadarVerificationManager.m

```diff
@@ -182,57 +182,47 @@ - (void)trackVerifiedWithBeacons:(BOOL)beacons completionHandler:(RadarTrackVeri
     }];
 }
 
-- (void)startTrackingVerifiedWithInterval:(NSTimeInterval)interval beacons:(BOOL)beacons {
```

Comment:
If I were starting fresh approaching this bugfix, I think I'd actually just opt for some state checks on `self.started` and `self.scheduled` (ie only start the loop if you weren't previously started).

I think the approach taken here works as well, but we should be cognizant of the bug ShiCheng pointed out re: stopping failing when a request is in flight: https://github.com/radarlabs/radar-sdk-ios/pull/406/files#r1812908221

Reply:
That was a good catch, fixed that

Intention here was to fix the double-start bug but also mirror the code on Android for consistency, so felt that it was worth making a slightly bigger change